### PR TITLE
Move new project tutorial to site

### PR DESCRIPTION
**Pull request by Dusch4593**
Closes issue 

- [x] My project tutorial topic has already been approved by the Codédex team
- [x] My project tutorial file renders properly in `mdx`
- [x] The content written is my own, and not plagiarized from other sources
- [x] All code written is commented or documented thoroughly
- [x] I have already completed a self-review of the project tutorial 
- [x] I will comply with changes from the Codédex team about my project tutorial

### CODE REVIEW

File: projects/build-a-pomodoro-app-with-html-css-js/index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          <div class="semi-circle right-side circle-mask"></div>
+          <div class="semi-circle right-side circle"></div>
+          <div class="semi-circle left-side circle-mask"></div>
+          <div class="semi-circle left-side circle"></div>
+        </div>
+        <div class="app-counter-box">
+          <p><span class="minutes">2</span>:<span class="seconds">00</span></p>
+          <!-- <div class="minutes">25</div>
+          <div>:</div>
+          <div class="seconds">00</div> -->
```
Comment (Goku-kun) [inline projects/build-a-pomodoro-app-with-html-css-js/index.html:25]: This sets the timer for 2 minutes. We should revert it back to 25 minutes. The text is overflowing with 25 minutes though.
image

**Review by Goku-kun — COMMENTED** [review]
We need to make some more changes before we publish this tutorial.

File: projects/build-a-pomodoro-app-with-html-css-js/build-a-pomodoro-app-with-html-css-js.mdx [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+This establishes the basic structure for our Pomodoro App.
+
+Next, let's add the following code to the `<head>` element:
+
+```html
+<head>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=EG+Garamond:wght@400;500;600;700&family=Fira+Sans:wght@400;500;600;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" ref="./style.css">
```
[the remark this answers is not in the document]
Comment (sonnynomnom) [reply projects/build-a-pomodoro-app-with-html-css-js/build-a-pomodoro-app-with-html-css-js.mdx:86]: +1. Let's take these out.

File: projects/build-a-pomodoro-app-with-html-css-js/build-a-pomodoro-app-with-html-css-js.mdx [hunk 3]
```diff
@@ -0,0 +1,452 @@
...
+  <title>Pomodoro App</title>
+</head>
+```
+
+We just added a few `<link>` elements:
+
+- Three `<link>` elements are for importing specific fonts from [Google](https://fonts.google.com/).
```
[the remark this answers is not in the document]
Comment (sonnynomnom) [reply projects/build-a-pomodoro-app-with-html-css-js/build-a-pomodoro-app-with-html-css-js.mdx:93]: yeah good catch

File: projects/build-a-pomodoro-app-with-html-css-js/index.html (hunk 1 above)
Comment (Dusch4593) [reply projects/build-a-pomodoro-app-with-html-css-js/index.html:25]: @Goku-kun Ah yes, this was why I originally commented out the original `<div>` copy and replaced with the `<p>` and `<span>` elements. This is what my screen shows when *_not_* using the commented edits:

Screenshot 2023-03-18 at 12 39 37 PM

File: projects/build-a-pomodoro-app-with-html-css-js/build-a-pomodoro-app-with-html-css-js.mdx [hunk 4]
```diff
@@ -0,0 +1,452 @@
...
+  - intermediate
+---
+
+## Introduction
+
+Welcome to the Pomodoro App coding tutorial! In this project, you'll learn how to create a simple and effective time management tool using just HTML, /CSS, and JavaScript. By the end of this tutorial, you'll have a basic understanding of how to build a dynamic web page that responds to user interactions. You'll also learn how to structure and style your HTML elements with CSS, and how to add interactivity with JavaScript. 
+
+The final project will look like this:
+
+<img src="./finished_pomodoro.gif" alt="Finished Pomodoro App" />
```
[the remark this answers is not in the document]
Comment (Dusch4593) [reply projects/build-a-pomodoro-app-with-html-css-js/build-a-pomodoro-app-with-html-css-js.mdx:20]: @Goku-kun Would it be `https://raw.githubusercontent.com/codedex-io/projects/main/projects/build-a-pomodoro-app-with-html-css-js/name-of-file`?

File: projects/build-a-pomodoro-app-with-html-css-js/app.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        clearInterval(myInterval);
+      }
+    }
+    myInterval = setInterval(updateSeconds, 1000);
+  } else {
+    alert('Session has already started.')
+  }
+}
+
+startBtn.addEventListener('click', appTimer);
```
Comment (Goku-kun) [inline projects/build-a-pomodoro-app-with-html-css-js/app.js:42]: Let's add a new line here.

File: projects/build-a-pomodoro-app-with-html-css-js/build-a-pomodoro-app-with-html-css-js.mdx [hunk 6]
```diff
@@ -0,0 +1,461 @@
...
+The styles for the `<html>` element set the font and background of our app.
+
+Next, let's add the following additional styles for our app headings and overall container:
+
+```css
+h1 {
+  margin: 0 auto 10px auto;
+  color: #d8e9ef;
+}
+
```
Comment (Goku-kun) [inline projects/build-a-pomodoro-app-with-html-css-js/build-a-pomodoro-app-with-html-css-js.mdx:188]: We need to add the following over here to match the file code:
```css
p {
  margin: 0;
}
```

**Pull request merged**